packages/api-client/src/connections/EntityApi.ts

```diff
@@ -14,12 +14,10 @@ import type {
   EntityResponseObject,
 } from '@tupaia/entity-server/src/type-exports';
 
-import { FilterCriteria, AdvancedFilterValue } from '../models';
+import type { FilterCriteria, AdvancedFilterValue } from '@tupaia/server-boilerplate/src/type-exports';
```

Comment:
Importing types from other packages is ok. They don't count towards being an npm dependency. 

However, these imported types are not present in the published package. So currently other users of this package do not get tupaia internal package typings. I looked at bundling them all up into a single .d.ts file, and some people do this, but it's not official or foolproof. So I went with this MVP solution.